packages/manager/cypress/e2e/core/account/account-logout.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { LOGIN_ROOT } from 'src/constants';
+import { interceptGetAccount } from 'support/intercepts/account';
+
+describe('Logout Test', () => {
+  beforeEach(() => {
+    cy.tag('purpose:syntheticTesting');
+  });
+
+  /*
+   * - Confirms that Cloud Manager log out functionality works as expected.
+   * - Confirms that the login application is up after account logout.
+   */
+  it('can logout the account and redirect to login endpoint', () => {
+    interceptGetAccount().as('getAccount');
+
+    cy.visitWithLogin('/account');
+    cy.wait('@getAccount');
+
+    // User can click Logout via user menu.
+    cy.findByTestId('nav-group-profile').click();
+    cy.findByTestId('menu-item-Log Out')
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+    // Upon clicking "Log Out", the user is redirected to the login endpoint at <REACT_APP_LOGIN_ROOT>/login
+    cy.url().should('equal', `${LOGIN_ROOT}/login`);
+    // Using cy.visit to navigate back to Cloud results in another redirect to the login page
+    cy.visit('/login');
+    cy.url().should('endWith', `/login`);
```

Comment:
```suggestion
    cy.visit('/');
    cy.url().should('startWith', `${LOGIN_ROOT}/login`);
```
Otherwise the URL assertion will immediately pass without waiting for Cloud to actually redirect to the Login application since both pages end with `/login`

Reply:
Updated, thank you!